### PR TITLE
Fix integration of compute_conflict through layers

### LINKED ISSUE #1203: `compute_conflict!` resulting in MethodError
I’m looking into `JuMP.compute_conflict!` to investigate an infeasiblity, however I get the following error. Note that the problem is infeasible. I’m using JuMP 0.21.5 and julia 1.4.2. @mlubin suggested that I open an issue.

```
julia> termination_status(model)
INFEASIBLE::TerminationStatusCode = 2
julia> compute_conflict!(model)
ERROR: MethodError: no method matching compute_conflict!(::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}})
Stacktrace:
 [1] compute_conflict!(::Model) at /Users/raphaelsaavedra/.julia/packages/JuMP/qhoVb/src/optimizer_interface.jl:161
 [2] top-level scope at REPL[48]:1
```

### PR DESCRIPTION
**Pull request by odow**
Closes #1203 

Still needs a test for the `is_bridged` branch.

### CODE REVIEW

File: src/Bridges/lazy_bridge_optimizer.jl [hunk 1]
```diff
@@ -385,3 +385,21 @@ function objective_functionize_bridge(b::LazyBridgeOptimizer)
...
+end
+
+function MOI.get(
+    model::LazyBridgeOptimizer,
+    ::MOI.ConstraintConflictStatus,
+    c::MOI.ConstraintIndex,
+)
+    if is_bridged(model, c)
+        error(
+            "Unable to query `ConstraintConflictStatus` of $(c) because the " *
```
Comment (blegat) [inline src/Bridges/lazy_bridge_optimizer.jl:400]: See https://github.com/jump-dev/MathOptInterface.jl/issues/1203#issuecomment-744303302

Comment (odow) [reply src/Bridges/lazy_bridge_optimizer.jl:400]: It seemed better to throw a nicer error message than:
```Julia
julia> MOI.get(model, MOI.ConstraintConflictStatus(), d)
ERROR: ArgumentError: Bridge of type `MathOptInterface.Bridges.Constraint.ScalarSlackBridge{Float64,MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}` does not support accessing the attribute `MathOptInterface.ConstraintConflictStatus()`.
Stacktrace:
 [1] get(::MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer}, ::MathOptInterface.ConstraintConflictStatus, ::MathOptInterface.Bridges.Constraint.ScalarSlackBridge{Float64,MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge.jl:61
 [2] (::MathOptInterface.Bridges.var"#54#55"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintConflictStatus})(::MathOptInterface.Bridges.Constraint.ScalarSlackBridge{Float64,MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge_optimizer.jl:920
 [3] (::MathOptInterface.Bridges.var"#1#2"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}},MathOptInterface.Bridges.var"#54#55"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintConflictStatus}})() at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge_optimizer.jl:242
 [4] call_in_context(::MathOptInterface.Bridges.Variable.Map, ::Int64, ::MathOptInterface.Bridges.var"#1#2"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}},MathOptInterface.Bridges.var"#54#55"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintConflictStatus}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/Variable/map.jl:404
 [5] call_in_context(::MathOptInterface.Bridges.Variable.Map, ::MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}, ::Function) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/Variable/map.jl:435
 [6] call_in_context(::MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer}, ::MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}, ::MathOptInterface.Bridges.var"#54#55"{MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer},MathOptInterface.ConstraintConflictStatus}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge_optimizer.jl:241
 [7] get(::MathOptInterface.Bridges.LazyBridgeOptimizer{Gurobi.Optimizer}, ::MathOptInterface.ConstraintConflictStatus, ::MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Bridges/bridge_optimizer.jl:920
 [8] get(::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, ::MathOptInterface.ConstraintConflictStatus, ::MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}}) at /Users/oscar/.julia/packages/MathOptInterface/ZJFKw/src/Utilities/cachingoptimizer.jl:605
 [9] _moi_get_result(::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, ::MathOptInterface.ConstraintConflictStatus, ::Vararg{Any,N} where N) at /Users/oscar/.julia/packages/JuMP/qhoVb/src/JuMP.jl:848
 [10] get(::Model, ::MathOptInterface.ConstraintConflictStatus, ::ConstraintRef{Model,MathOptInterface.ConstraintIndex{MathOptInterface.ScalarAffineFunction{Float64},MathOptInterface.Interval{Float64}},ScalarShape}) at /Users/oscar/.julia/packages/JuMP/qhoVb/src/JuMP.jl:887
 [11] top-level scope at REPL[19]:1
```

The default error will just cause people to file issues.

Comment (blegat) [reply src/Bridges/lazy_bridge_optimizer.jl:400]: We could say that for any attribute though, how is this one specific ? It also does not seem unreasonable to implement it for several bridges so the error message should mention that the issue is that a specific bridge type does not implement it and that's exactly what the error message is saying here. Maybe we could improve the error message for all attribute then if you feel it's not clear.

Comment (odow) [reply src/Bridges/lazy_bridge_optimizer.jl]: I've removed the specific error. We can either improve the error message or implement `ConstraintConflictStatus` for bridges.

**Review by joaquimg — APPROVED** [review]

File: src/Bridges/lazy_bridge_optimizer.jl (hunk 1 above)
Comment (odow) [reply src/Bridges/lazy_bridge_optimizer.jl]: Opened an issue to track: https://github.com/jump-dev/MathOptInterface.jl/issues/1221

**Pull request merged**